cmd/sourced/cmd/sql.go

```diff
@@ -8,10 +8,19 @@ import (
 
 type sqlCmd struct {
 	Command `name:"sql" short-description:"Open a MySQL client connected to a SQL interface for Git" long-description:"Open a MySQL client connected to a SQL interface for Git"`
+
+	Args struct {
+		Query string `positional-arg-name:"query" description:"SQL query to be run by the SQL interface for Git"`
+	} `positional-args:"yes"`
 }
 
 func (c *sqlCmd) Execute(args []string) error {
-	return compose.Run(context.Background(), "exec", "gitbase", "mysql")
+	command := []string{"exec", "gitbase", "mysql"}
+	if query := c.Args.Query; query != "" {
```

Comment:
why not use `c.Args.Query` directly, and skip the `query` var?

Reply:
Yes, it's cleaner that other way, thanks! ;)
pushforced